controllers/home-routes.js

```diff
@@ -10,4 +10,7 @@ router.get('/', withAuth, (req, res) => {
   res.render('homepage');
 })
 
+router.get('/sign-up', (req,res) => {

```

Comment:
I noticed that in the `login.handlebars` file, the `href` leads to `/signup` rather than `/sign-up`, which lead to getting a "cannot get /signup" error. This should be an easy fix, just change the href (I believe it's on line 44 of `login.handlebars`) to `/sign-up`